datasets/loopfietsnetwerk/dataset.json

```diff
@@ -0,0 +1,283 @@
+{
+    "type": "dataset",
+    "id": "loopfietsnetwerk",
+    "title": "loopfietsnetwerk",
+    "description": "Topologisch consistent netwerk bedoeld voor het uitvoeren van ruimtelijke netwerkanalyses in het gemeentelijke domein",
+    "spatialDescription": "Gemeente Amsterdam, inclusief Weesp",
+    "license": "public",
+    "status": "beschikbaar",
+    "version": "0.0.1",
+    "theme": [
+      "Ruimte en Topografie",
+      "Verkeer",
+      "Wonen"
+    ],
+    "owner": "Gemeente Amsterdam",
+    "auth": "OPENBAAR",
+    "authorizationGrantor": "Onderzoek & Statistiek",
+    "creator": "Onderzoek & Statistiek",
```

Comment:
Onder `creator` kan de bronhouder (eigenaar van de data) ingevuld worden. Is de bronhouder van het loopfietsnetwerk bekend, specifieker dan O&S?

Reply:
Deze dataset is omgebouwd verschillende bronnen, waaronder Openstreetmap. We zien onszelf als verrijker..
Voorstel is om dit datateam als bronhouder aan te houden.